### ISSUE TITLE
In the Trace panel, the query started to go backwards after each request

### ISSUE THREAD

**Issue by flymedllva** [wundergraph/cosmo#678]
### Component(s)

router

### Component version

v0.78.0

### wgc version

0.49.1

### controlplane version

-

### router version

-

### What happened?

After some recent update, in the `Trace` panel in playground, the query started to go backwards after each request

![Screenshot 2024-04-02 at 14 04 47](https://github.com/wundergraph/cosmo/assets/33265875/9f6fe1c6-814e-4717-83c8-a9d4fef49b76)
![Screenshot 2024-04-02 at 14 04 47](https://github.com/wundergraph/cosmo/assets/33265875/14a00a49-c75a-4c5c-bc96-a6ca112da19c)
![Screenshot 2024-04-02 at 14 04 55](https://github.com/wundergraph/cosmo/assets/33265875/3dcad96c-a1f9-4f7d-ae3c-103d330027ec)

**Comment by Aenimus** [comment]
Hi,

We're really not sure what you mean by "started to go backwards". Could you elaborate on the issue?

Is this simply a visual issue?

Thanks!

**Comment by flymedllva** [comment]
Yes it is a visual problem, but it interferes with working with this panel. Every new request with debug true, moves the whole interface down, if you make 10 requests, you will not be able to see where the requests went at all, I am playing on a 2k monitor.

**Comment by Aenimus** [comment]
That makes it much clearer, thanks!

**Comment by endigma** [comment]
Hi @flymedllva , I'm closing this as stale, feel free to reopen if you're still experiencing this issue!
